src/Dfe.Spi.Common/Dfe.Spi.Common.Caching/Definitions/Caches/ICacheProvider{TCacheKey,TCacheValue}.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Dfe.Spi.Common.Caching.Definitions.Caches
+{
+    /// <summary>
+    /// Describes the operations of the cache provider.
+    /// </summary>
+    /// <typeparam name="TCacheKey">
+    /// The type of key used in the underlying storage.
+    /// </typeparam>
+    /// <typeparam name="TCacheValue">
+    /// The type of item to store in the cache.
+    /// </typeparam>
+    public interface ICacheProvider<TCacheKey, TCacheValue>
```

Comment:
Do these generics provide value? Could the methods not just assume key is string and value is object?

Reply:
Code updated to use strings and objects, as suggested.